src/Features/Protocol/LanguageServices/Extensions/RoslynToProtocolExtensions.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using VSLocation = Microsoft.VisualStudio.LanguageServer.Protocol.Location;
```

Comment:
>VSLocation [](start = 6, length = 10)

I'd rather use  `Protocol` or `Lsp` prefix than `VS`. E.g. `ProtocolLocation` or `LspLocation`.

Reply:
Actually, even better approach would be to define `using Lsp = Microsoft.VisualStudio.LanguageServer.Protocol;` and then refer to all types in the namespace as `Lsp.Xxx`, etc. `Lsp.Location`.

---
In reply to: [274065235](https://github.com/dotnet/roslyn/pull/34794#discussion_r274065235) [](ancestors = 274065235)